docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Observability_Tracing.adoc

```diff
@@ -0,0 +1,102 @@
+[[Observability_Tracing]]
+= OpenTelemetry Subsystem Configuration
+
+[[required-extension-opentelemetry]]
+== Extension
+
+This extension is included in all the standalone configurations included in the WildFly distribution.
+
+== Configuration
+
+Systems administrators can configure a number of aspects of OpenTelemetry: the exporter, span processor, and sampler.
+
+=== Exporter
+
+The exporter can be selected and configured using the `exporter` child element, which supports these attributes:
+
+* `exporter`: WildFly supports two different exporters:
+** `jaeger`: The default exporter
+** `otlp`: The OpenTelemetry protocol
+* `endpoint`: The URL via which OpenTelemetry will push traces. The default is Jaeger's gRPC-based endpoint,
+http://localhost:14250
+
+=== Span Processor
+The span process is configured via the `span-processor` element, which supports the following attributes:
+
+* `type`: The type of span processor to use.
+** `batch`: The default processor, which sends traces in batches as configured via the remaining attributes
+** `simple`: Traces are pushed to the exporter as they finish.
+* `batch-delay`: The amount of time, in milliseconds, to wait before traces are published (default: 5000)
+* `max-queue-size`: The maximum size of the queue before traces are dropped (default: 2048)
+* `max-export-batch-size`: The maximum number of traces that are published in each batch, which must be smaller or equal
+to `max-queue-size (default: 512)
+* `export-timeout`: The maximum amount of time in milliseconds to allow for an export to complete before being cancelled
+(default: 30000)
+
+=== Sampler
+
+The sampler is configured via the `sampler` element:
+
+* `type`: The type of sampler to use
+** `on`: Always on (all traces are recorded)
+** `off`: Always off (no traces are recorded)
+** `ratio`: Return a ratio of the traces (e.g., 1 trace in 10000).
+* `sampler-arg`: The value use to configure the `ratio` sampler, which must be within [0.0, 1.0]. For example, if 1
+trace in 10,000 is to recorded, this value would be `0.0001`.
+
+=== Example Configuration
+
+The following XML is an example of the full configuration, including default values (WildFly does not typically persist
+default values, so what you see in the configuration file may look different):
+
+[code,xml]
+-----
+<subsystem xmlns="urn:wildfly:opentelemetry:1.0" service-name="example">
```

Comment:
What about the `service-name` attribute? Is it configurable? Could you add it to the rest of the configurability information?

Reply:
This *should* work now, though I still need to verify locally.

Reply:
Confirmed.